cores/cv32e40p/user_manual/source/control_status_registers.rst

```diff
@@ -9,135 +9,149 @@ that were needed for the PULP system. The reason for this is that we
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| 0x001         | ``fflags``        | URW       | ``FPU`` = 1         | Floating-point accrued exceptions.                      |
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| 0x002         | ``frm``           | URW       | ``FPU`` = 1         | Floating-point dynamic rounding mode.                   |
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| 0x003         | ``fcsr``          | URW       | ``FPU`` = 1         | Floating-point control and status register.             |
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| User Custom CSRs                                                                                                              |
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| 0x7C0         | ``lpstart0``      | URW       | ``PULP_XPULP`` = 1  | Hardware Loop 0 Start.                                  |
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| 0x7C1         | ``lpend0``        | URW       | ``PULP_XPULP`` = 1  | Hardware Loop 0 End.                                    |
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| 0x7C2         | ``lpcount0``      | URW       | ``PULP_XPULP`` = 1  | Hardware Loop 0 Counter.                                |
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| 0x7C4         | ``lpstart1``      | URW       | ``PULP_XPULP`` = 1  | Hardware Loop 1 Start.                                  |
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| 0x7C5         | ``lpend1``        | URW       | ``PULP_XPULP`` = 1  | Hardware Loop 1 End.                                    |
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| 0x7C6         | ``lpcount1``      | URW       | ``PULP_XPULP`` = 1  | Hardware Loop 1 Counter.                                |
++---------------+-------------------+-----------+---------------------+---------------------------------------------------------+
+| 0x006         | ``fprec``         | URW       | ``PULP_XPULP`` = 1  | Custom flag which controls the precision and latency    |
```

Comment:
Condition is FPU = 1 && PULP_XPULP =1 (as correctly documented at the fprec description itself)

Reply:
Fixed.